.github/ISSUE_TEMPLATE/3.pro-support.yml

```diff
@@ -9,31 +9,39 @@ body:
         Please provide a searchable summary of the issue in the title above ⬆️.
 
         ⚠️ **Don't use this form if the problem is [a bug](https://github.com/mui/mui-x/issues/new?assignees=&labels=status%3A+needs+triage&template=1.bug.yml) or a [feature request](https://github.com/mui/mui-x/issues/new?assignees=&labels=status%3A+needs+triage&template=2.feature.yml), use the dedicated form instead.**
+
   - type: input
     id: contact
     attributes:
-      label: Order ID or Support key 💳 (optional)
+      label: Order ID or Support key 💳
       description: The order ID of the purchased Pro plan. Community users can [learn more about support](https://mui.com/getting-started/support/) in the documentation.
       placeholder: 'e.g. 11111'
     validations:
       required: true
-  - type: checkboxes
+
+  - type: input
     attributes:
-      label: Duplicates
-      description: Please [search the history](https://github.com/mui/mui-x/issues) to see if an issue already exists for the same problem.
-      options:
-        - label: I have searched the existing issues
-          required: true
+      label: Search keywords
+      description: Your issue may have already been reported! List the keywords you've used to search the [existing issues](https://github.com/mui/mui-x/issues). This will also make your issue searchable for others.
+      placeholder: e.g. datagrid column resizing
+    validations:
+      required: true
   - type: checkboxes
     attributes:
       label: Latest version
       description: We roll bug fixes, performance enhancements, and other improvements into new releases.
       options:
         - label: I have tested the latest version
           required: true
+
   - type: textarea
     attributes:
       label: The problem in depth 🔍
+      description: |
+        **If you're reporting a bug, please provide a live example for your report.**
+        If you don't have one, you can use one of these options:
+          - [DataGrid codesandbox template](https://codesandbox.io/s/github/mui/mui-x/tree/master/templates/x-data-grid?file=/demo.tsx)
+          - Fork any of the examples in our [documentation](https://mui.com/x/introduction/) by [clicking on the codesandbox or stackblitz icon](https://mui.com/static/docs/forking-an-example.png)
```

Comment:
I'm adding this because it happens regularly that the pro/premium support templates are used to submit a bug report, and it's rare that we get a clear reproduction case unless we explicitly ask for one.

Example: https://github.com/mui/mui-x/issues/10404